example/basic-redis.js

```diff
@@ -0,0 +1,30 @@
+import Redis from 'ioredis'
+import Fastify from 'fastify'
+import slowDownPlugin from '../index.js'
+
+let id = 0
```

Comment:
what's this id for?

Reply:
Just to make sure that connection name is different between two servers

Reply:
Can't we use something like `Date.now()`?

Reply:
yeah let's avoid using this mutable state approach that's quite ugly

Reply:
or just use the port really